ci/setup.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
Could we also update README.md and/or CONTRIBUTING.md file to mention the script?

Reply:
Yeah I've opened this PR to see suggestions where it can be added (readme doesn't fit well as for me, I'd put to contributing.md)